bgpd/bgp_network.c

```diff
@@ -203,7 +203,7 @@ int bgp_set_socket_ttl(struct peer *peer, int bgp_sock)
 	int ret = 0;
 
 	/* In case of peer is EBGP, we should set TTL for this connection.  */
-	if (!peer->gtsm_hops && (peer_sort(peer) == BGP_PEER_EBGP)) {
+	if (!peer->gtsm_hops && (peer_sort_lookup(peer) == BGP_PEER_EBGP)) {
```

Comment:
there is assumption here that peer->sort is already set. If not already set then we may be using wrong value?

Reply:
This is a backport to 7.3 to fix compilation issues:
```
bgpd/bgpd.c: In function ‘peer_as_change’:
bgpd/bgpd.c:1588:13: warning: implicit declaration of function ‘peer_sort_lookup’ [-Wimplicit-function-declaration]
  origtype = peer_sort_lookup(peer);
             ^~~~~~~~~~~~~~~~
```